stl/inc/format

```diff
@@ -385,15 +386,106 @@ _NODISCARD constexpr const _CharT* _Parse_arg_id(
     throw format_error("Invalid format string.");
 }
 
+inline int _Code_units_in_next_character(const char* _First, const char* _Last, const _Cvtvec& _Cvt) {
+    // Returns a count of the number of code units (characters) that compose the first multibyte
+    // character in [_First, _Last), or -1 if [_First, _Last) doesn't contain an entire encoded
+    // character or *_First is not a valid lead byte.
+    _STL_INTERNAL_CHECK(_First < _Last);
+
+    switch (_Cvt._Mbcurmax) {
+    default:
+        _ASSERTE(!"Bad number of encoding units for this code page");

```

Comment:
We haven't used `_ASSERTE` in headers before. Should this be `_STL_ASSERT(false, "message");`?

Reply:
I think I'll simply `_STL_INTERNAL_CHECK` this.